### ISSUE TITLE
Compatibility Issue With Custom Ores

### ISSUE THREAD

**Issue by Lezreth** [Nanabell/Loader-Furnaces#1]
The change from an actual furnace to something more resembling an assembler in your newest version broke compatibility with custom ores, like pulverized iron or steel dust from Hard Crafting.

**Comment by Nanabell** [comment]
that is true and I feared that this would become an issue. The reason I changed it to an assembler is pretty simple, the last tier of Loader Furnaces (Extremly Fast Loader Furnace) has to craft 66.666 items per second which is impossible if you have a crafting result of 1 per second as the game only runs at 60ups so you are missing .666 items per second on the belt and it will never be fully compressed, if you check the recipes now they all produce 2x items and have a generlized crafing speed to ensure that you will alaways get a full belt output of your loader type.

I could simply allow the Loader Furnaces to smelt normal smelting recipes but with that, i cant guarantee a full belt as the crafting time might differ

**Comment by Nanabell** [comment]
Update
I've updated the mod to allow to work with normal smelting recipes since that seemed to most sensible approach. The only limitation that could be causing issues if a mod author chose to make his smelting recipes hidden then you cant select them.
 But as I said before most recipes won't result in a full belt of resources

**Comment by Lezreth** [comment]
I ran various tests with the updated mod and came out with some interesting results.

- Steel Dust may not result in fully compressed steel plates, but neither does the vanilla recipe.
      - But Steel Dust is more compressed than vanilla.
- The other Hard Crafting recipes that produce just the plates work really well, producing fully compressed belts.
- The Hard Crafting recipes that produce byproducts and have longer crafting times are not fully compressed, but are sorted when they come out.  This makes further processing easier. (Copper is different, but that's a problem with the recipe.)
- The recipes with shorter times but produce byproducts are fully compressed _and_ sorted.

It is a little disappointing that I have to change the recipe every time I switch things around, but overall I am satisfied with the results.

**Comment by Nanabell** [comment]
>Steel Dust may not result in fully compressed steel plates, but neither does the vanilla recipe.
 But Steel Dust is more compressed than vanilla.

I think this is likely my oversight that bob's changes the default recipe from 5to1 to 2to1 in steel production, I will update that in the next update.

I was also thinking about ways to turn the now Loader Furnace Assembling Machine back into an actual furnace but haven't found a good solution for now,

I plan to also add support for Hard Crafting in general but I can't say when I will get to that as I have quite a bit of stuff in real life atm. 
I would like leave this issue open until I've added full support for HardCrafting

**Comment by Karmacopper** [comment]
TO add to this rather than raising a new issue. Incompatible with https://mods.factorio.com/mods/toastii/OreCrusher

**Comment by Nanabell** [comment]
@Karmacopper please open a separate issue for OreCrusher support so i don't forget about it when i get some free time

**Comment by shadoxxhd** [comment]
Would it be possible to have multiple furnaces in one entity?
